about/distribution/index.markdown

```diff
@@ -0,0 +1,80 @@
+---
+author: admin
+comments: false
+date: 2022-02-03
+layout: page
+slug: distribution
+title: Distribution
+---
+
+<div class='row'>
+  <div class='col-sm-12 text-center'>
+    <h1>Distributions</h1>
+    <p class="paragraph-big">
+     You're reading the documentation for a version of ROS 2 that has reached its EOL (end-of-life), and is no longer officially supported. If you want up-to-date information, please have a look at Galactic.
+    </p>
+  </div>
+  <div class="col-sm-12">
+    <h2>What is a Distribution?</h2>
+    <a href="https://wiki.ros.org/Distributions" target="_blank">See wiki.ros.org/Distributions.</a>
+    <h2>List of Distributions</h2>
+    <table class="table table-striped table--distribution">
+      <thead>
+        <tr>
+          <th>Distro</th>
+          <th>Release date</th>
+          <th>Logo</th>
+          <th>EOL date</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td><a href="https://docs.ros.org/en/crystal/Releases/Release-Foxy-Fitzroy.html">MoveIt Rolling</a></td>
+          <td>June 5th, 2020</td>
+          <td></td>
+          <td>N/A</td>
+        </tr>
+
+        <tr>
+          <td><a href="https://moveit.ros.org/ros/moveit/galactic/2021/07/08/moveit-galactic.html">MoveIt 2 Galactic</a></td>
+          <td>June 5th, 2020</td>
+          <td><img src="/assets/images/distribution/galactic.png" height="100" /></td>
+          <td>May 2023</td>
```

Comment:
Shouldn't we just use the same EOL dates as the ROS 2 distros? That would be November 2022 for Galactic and May 2023 for Foxy

Reply:
ok I'll update them.